tests/functional/cylc-install/04-symlinks-not-resolved.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#------------------------------------------------------------------------------
+# Check that symlink is not resolved to its target on installation:
+# When ~/foo -> ~/bar and --symlink-dirs=run=~/bar DO NOT make run=~/foo
+
+. "$(dirname "$0")/test_header"
+set_test_number 3
+
+cat > flow.cylc <<__HEREDOC__
+[scheduler]
+    allow implicit tasks = True
+[scheduling]
+    initial cycle point = 1500
+    [[graph]]
+        R1 = foo
+__HEREDOC__
+
+run_ok "cylc validate ."
+
+# Create a temporary directory to put our symlinked foo and bar in:
+elsewhere=$(mktemp -d)
```

Comment:
My bash inexperience here so happy to be corrected, should this `elsewhere` be in capitals?

Reply:
Bash doesn't mandate it. It's just common practice.